Makefile

```diff
@@ -13,7 +13,7 @@ lint:
 	cargo fmt --check
 	cargo test -p kamu-repo-tools
 	cargo deny check
-	cargo clippy all
+	cargo clippy --all --all-targets
```

Comment:
The docs on `cargo check` say switch `--all` should be replaced with `--workspace`:

```
 --all               Alias for --workspace (deprecated)
 ```

Reply:
a good catch -- replaced